modules/ROOT/pages/mergetags.adoc

```diff
@@ -0,0 +1,54 @@
+= Merge Tags plugin
+:navtitle: Merge Tags
+:description: Enables Merge Tags functionality in the {productname} editor
+:keywords: merge, tags, tag, mergetags, mergetag, mergetags_list, mergetags_prefix, mergetags_suffix
```

Comment:
Should this include `mailmerge` or similar? We list it below as something it's commonly known as so that should help with SEO if we add it here.

Reply:
Added *mail merge* and *personalization token* as keywords.